.github/workflows/publish-environment.yml

```diff
@@ -23,6 +23,10 @@ jobs:
         uses: MetaMask/action-checkout-and-setup@v1
         with:
           is-high-risk-environment: true
+      - name: Restore build artifacts
```

Comment:
I guess we could also re-build everything here. WDYT @Mrtenz ?

Reply:
Let's do that, since this workflow can technically be used outside of the release workflow.